autodarts-caller.py

```diff
@@ -838,20 +844,61 @@ def mirror_sounds():
         broadcast(mirror)
         mirror_files = []
 
+def walk_on(playersInMatch):
+    global waitForWalkOns
+    if WALK_ON and playersInMatch[0]["name"] != None:
```

Comment:
use loops, instead of repeating yourself.

Reply:
yes sir.